lib/asserts.nix

```
@@ -50,4 +50,33 @@ rec {
       lib.generators.toPretty {} xs}, but is: ${
         lib.generators.toPretty {} val}";
 
+  /* Specialized `assertMsg` for checking if every one of `vals` is one of the elements
+     of the list `xs`. Useful for checking lists of supported attributes.
+
+     Example:
+       let sslLibraries = [ "libressl" "bearssl" ];
+       in assertOneOf "sslLibraries" sslLibraries [ "openssl" "bearssl" ]
```

Am I reading this right?

```suggestion
       in assertEachOneOf "sslLibraries" sslLibraries [ "openssl" "bearssl" ]
```

---

yeah that’s what they call a copy & paste error in the biz :P

---

I also fixed the type in the docstring